### PR TITLE
D8NID-1189 CSP module and config

### PR DESCRIPTION
**Pull request by johangant**
* Adds the content security policy module with report only config.
* Adds filelog module to avoid spraying a firehose of events at dblog (platform.sh can't support syslog)
* Cron handler to prune away compressed log files over 30 days old.

### CODE REVIEW

**Review by neilblair — CHANGES_REQUESTED** [review]
Looks good Johan - only concern is frame-ancestors settings.  I think we'll need that set to 
`frame-ancestors: 'self' www.youtube.com` or something like that?  Unless that is oembed works ok without that?

File: config/sync/csp.settings.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        - www.googletagmanager.com
+      base: self
+    style-src:
+      base: self
+    style-src-attr:
+      flags:
+        - unsafe-inline
+      base: self
+    frame-ancestors:
+      base: any
```
Comment (neilblair) [inline config/sync/csp.settings.yml:69]: Think frame-ancestors should be self - and might need to specify youtube domain(s) as sources

**Review by neilblair — CHANGES_REQUESTED** [review]

**Review by DuttonMa — APPROVED** [review]
Interesting stuff !

File: config/sync/csp.settings.yml (hunk 1 above)
Comment (johangant) [reply config/sync/csp.settings.yml:69]: Yes, thanks for spotting. I had taken it off earlier to test something and forgotten to reset it. Sorted now

**Comment by johangant** [comment]
Addressed earlier issue raised by @neilblair - merging so it's ready for Monday

**Pull request merged**